src/keria/app/agenting.py

```diff
@@ -745,23 +746,24 @@ def __init__(self, kvy, rgy, rvy, tvy, exc, vry, registrar, credentialer):
         self.vry = vry
         self.registrar = registrar
         self.credentialer = credentialer
+        self.tock = 1.0
 
-        super(Escrower, self).__init__()
+        super(Escrower, self).__init__(tock=self.tock)
 
-    def recur(self, tyme):
+    def recur(self, tyme=None):
         """ Process all escrows once per loop. """
-        self.kvy.processEscrows()
-        self.kvy.processEscrowDelegables()
-        self.rgy.processEscrows()
-        self.rvy.processEscrowReply()
-        if self.tvy is not None:
-            self.tvy.processEscrows()
-        self.exc.processEscrow()
-        self.vry.processEscrows()
-        self.registrar.processEscrows()
-        self.credentialer.processEscrows()
-
-        return False
+        while True:
```

Comment:
This change is not needed.  Once a Doer has a self.tock value, the parent Doist/DoDoer will use that so you don't need to turn recur into a generator just to yield the tock.

Reply:
I didn't notice that than you for your feedback. I've updated the PR.